tests/types/src/valid.ts

```diff
@@ -596,6 +596,8 @@ addressElement.update({
   },
 });
 
+addressElement.getValue();
```

Comment:
Could we add a `.then()` here, like in https://github.com/stripe/stripe-js/blob/63773b2c774efb731188d1c5229cc0aa92b4e05e/tests/types/src/valid.ts#L2528-L2538?